components/Carousel/Carousel.js

```diff
@@ -1,14 +1,55 @@
 class Carousel {
+    constructor(element)    {
+        this.carousel = element;
+        this.buttonLeft = element.querySelector(".left-button");
+        this.buttonRight = element.querySelector(".right-button");
+        this.imgArray = element.querySelectorAll("img");
+        this.currentIndex = 0;
+        this.imgArray[0].style.display = "block";
+        this.clicked = false;
+        this.buttonLeft.addEventListener("click",   ()  =>  this.pictureLeft());
+        this.buttonLeft.addEventListener("click",   ()  =>  this.clicked = true);
+        this.buttonRight.addEventListener("click",  ()  =>  this.pictureRight());
+        this.buttonRight.addEventListener("click",  ()  =>  this.clicked = true);
```

Comment:
Could possibly combine `pictureLeft` with one of the `this.clicked` and same with `pictureRight`, but this is fine since it's more explicit.